Emby.Server.Implementations/Configuration/ServerConfigurationManager.cs

```diff
@@ -17,18 +18,55 @@ namespace Emby.Server.Implementations.Configuration
     /// </summary>
     public class ServerConfigurationManager : BaseConfigurationManager, IServerConfigurationManager
     {
+        private readonly IConfiguration _startupConfig;
+
         /// <summary>
         /// Initializes a new instance of the <see cref="ServerConfigurationManager" /> class.
         /// </summary>
         /// <param name="applicationPaths">The application paths.</param>
         /// <param name="loggerFactory">The logger factory.</param>
         /// <param name="xmlSerializer">The XML serializer.</param>
+        /// <param name="startupConfig">The startup configuration containing environment variables.</param>
         public ServerConfigurationManager(
             IApplicationPaths applicationPaths,
             ILoggerFactory loggerFactory,
-            IXmlSerializer xmlSerializer)
+            IXmlSerializer xmlSerializer,
+            IConfiguration startupConfig)
             : base(applicationPaths, loggerFactory, xmlSerializer)
         {
+            _startupConfig = startupConfig;
+
+            // Check for environment variable override AFTER base constructor loads XML
+            // You can enable metrics by setting the JELLYFIN_EnableMetrics environmental variable
```

Comment:
I think the convention, at least on Linux, is that environment variables are `ALL_CAPS_ALL_THE_WAY=true`

Reply:
Hi @jpds - I agree for env vars on Linux this is typically the convention; however, for this project I noticed that the env vars use `CamelCase`. I'm assuming it to be a Windows/C# convention

If any project maintainers can chime in, I'd be happy to make any required changes.